tests/unit/bases/test_ubuntu_buildd.py

```diff
@@ -686,6 +685,34 @@ def test_ensure_image_version_compatible_failure(fake_executor, monkeypatch):
     )
 
 
+@pytest.mark.parametrize("alias", list(ubuntu.BuilddBaseAlias))
+@pytest.mark.parametrize("cache_path", [pathlib.Path("/tmp/fake-cache-dir")])
+def test_mount_cache_dirs(fake_process, fake_executor, cache_path, alias):
+    """Test mounting of cache directories with a cache directory set."""
+    base = ubuntu.BuilddBase(alias=alias, cache_path=cache_path)
+    user_cache_dir = pathlib.Path("/root/.cache")
+    fake_process.register(
+        [*DEFAULT_FAKE_CMD, "bash", "-c", "echo -n ${XDG_CACHE_HOME:-${HOME}/.cache}"],
+        stdout=str(user_cache_dir),
+    )
+    fake_process.register(
+        [*DEFAULT_FAKE_CMD, "mkdir", "-p", "/root/.cache/pip"],
+    )
+
+    base._mount_shared_cache_dirs(fake_executor)
+
+    expected_mounts = [
+        {
+            "host_source": cache_path
+            / base.compatibility_tag
+            / str(base.alias)
+            / "pip",
```

Comment:
This isn't easy to read. Can we make `host_source` its own variable above so it's clearer?